app/profile/[address]/hypercerts-tab-content.tsx

```diff
@@ -27,7 +30,27 @@ const HypercertsTabContentInner = async ({
   const claimableHypercerts = await getAllowListRecordsForAddressByClaimed(
     address,
     false,
-  );
+  ).then(async (res) => {
+    if (!res?.data) {
+      return {
+        data: [],
+        count: 0,
+      };
+    }
+    const hypercertsWithMetadata = await Promise.all(
+      res.data.map(async (record): Promise<UnclaimedFraction> => {
+        const hypercert = await getHypercert(record.hypercert_id as string);
+        return {
+          ...record,
+          metadata: hypercert?.metadata || null,
```

Comment:
It's probably faster to directly query the metadata based on the hypercert ID, it skips calls for things like orders and sales, saving a lot of roundtrips:

`{
  metadata(where: {hypercerts: {hypercert_id: {eq: ""}}})
}
`